### DISCUSSION TITLE
Notifications on Dashboard

### DISCUSSION THREAD

**Discussion by nrdbstrd** [AprilSylph/XKit-Rewritten#392] [Suggestion Box]
Are there any plans to include a feature to show notes from the activity page on the dashboard? I'm not sure if it's still possible to place them in-line with posts like they used to be after the most recent activity page changes, but even something like a mini activity feed as a sidebar would be nice. While I'm fantasizing, it would also be nice to be able to choose which notifications show up on the dashboard, i.e. likes/reblogs/mentions, only from followers/mutuals, etc, and maybe an option to show tags with reblog notifications.

**Comment by AprilSylph** [comment]
It's not really feasible.
